Makefile

```diff
@@ -269,14 +270,13 @@ pytest_all:
 	@for i in $$(find projects -name "test_*.py"); 													\
 	do																								\
 		python -m unittest discover -t $$(dirname $$i) -s $$(dirname $$i) -p $$(basename $$i);		\
-	done			
+	done		
 
 .PHONY: install_requirements
 install_requirements:
-	@for i in $$(find projects -name "requirements.txt"); 		\
-	do															\
-		pip install -r $$i;										\
-	done								
+	@rm -rf .venv
+	@virtualenv .venv
```

Comment:
When I run `make install_requirements` on my system I get `virtualenv: Command not found` - maybe use `python3 -m venv` as it seems more portable.

Reply:
Interesting, I'm getting the opposite error "must install python3-venv". Should I write a note saying that you may need to run "sudo apt-get install python3-venv" or "sudo pip3 install virtualenv"?

Reply:
Yeah sure, also put that in the Software 101 setup on confluence.